crates/bevy_ui/src/widget/image.rs

```diff
@@ -1,9 +1,10 @@
 use crate::{measurement::AvailableSpace, ContentSize, Measure, Node, UiImage, UiScale};
 use bevy_asset::Assets;
 
-use bevy_ecs::change_detection::DetectChanges;
+#[cfg(feature = "bevy_text")]
```

Comment:
Using a `bevy_ecs::prelude::*` import should eliminate the need for this config and reduce merge conflicts in the future.

Reply:
Fixed in 5ce89049cac82d79c25ff40f39838185481739a5.